products/template/build_scripts/transform.py

```diff
@@ -1,16 +1,41 @@
+from dbt.cli.main import dbtRunner, dbtRunnerResult
+from dbt.contracts.results import RunExecutionResult
+
 from dcpy.utils import postgres
 from dcpy.utils.logging import logger
 from build_scripts import SQL_QUERY_DIR, PG_CLIENT
 
 
 def _run_step(step_query=str):
-    logger.info(f"Running transform script {step_query}")
+    logger.info(f"Running transform step {step_query} ...")
     postgres.execute_file_via_shell(
         PG_CLIENT.engine_uri,
         SQL_QUERY_DIR / f"{step_query}.sql",
     )
 
 
+def _run_dbt_step(run_args=list[str]):
+    dbt_cli_call = f"dbt {' '.join(run_args)}"
+    logger.info(f"Running '{dbt_cli_call}' ...")
+    dbt_runner = dbtRunner()
+    runner_result: dbtRunnerResult = dbt_runner.invoke(run_args)
```

Comment:
this is where dbt is called, see docs [here](https://docs.getdbt.com/reference/programmatic-invocations)

Reply:
is this command equivalent to `dbt run` CLI command?

Reply:
only if run is passed as the first `run_args`. so  `_run_dbt_step(["debug"])` is equivalent to `dbt debug`

I chose to align with the `_run_step` function above but maybe this shouldn't have the word "run" in the name?

Reply:
Got it! I think "run" in the fn name is a bit confusing given that there is a command with that keyword. How about `_execute_dbt_step` or `_perform_dbt_step`?